content/posts/xapi-activities-101/index.md

```diff
@@ -0,0 +1,262 @@
+---
+title: "xAPI Activities 101"
+date: 2023-07-04T10:13:24Z
+description:
+  "In this blog post we’ll dissect the xAPI specification and focus on the concept of activity. What
+  is an activity, where can we expect to find activity metadata and how are the different activity
+  metadata source related? Let's find out!"
+categories: ["xAPI", "Activity", "Activity Definition", "Activity Metadata", "LRS"]
+draft: false
+authors:
+  - "thibaultlebrun"
+---
+
+Activities are a backbone of the xAPI specification and of learning systems across the universe 💫.
```

Comment:
?

```suggestion
Activities are the backbone of the xAPI specification and of learning systems across the universe 💫.
```

Reply:
I meant the 'a' because there are other backbones of xAPI like agents or verbs .. ?